pkg/instance/handler.go

```diff
@@ -466,7 +468,7 @@ func (h Handler) Logs(c *gin.Context) {
 		_ = c.Error(err)
 	}
 
-	readCloser, err := h.instanceService.Logs(instance, group)
+	readCloser, err := h.instanceService.Logs(instance, group, selector)
```

Comment:
I would rename `readCloser` to `r` and any writer to `w` as this is how its usually done in the standard library and other Go projects.